ui/src/hooks/useZsentry.ts

```diff
@@ -126,9 +126,13 @@ export const useSentry = (lastSendTime?: Ref<Date>, t?: any): IUseSentry => {
             },
             onNegativeClick: () => {
                 if (zmodeSession.value) {
-                    debug('Cancel Abort');
-                    zmodeSession.value.abort();
+                    try {
+                        zmodeSession.value.abort();
+                    } catch (e) {
+                        return true;
+                    }
                 }
+                return true;
             }
         };
     });
```

Comment:
Since this question refers to code from before the date of completion, there is no specific comparison needed between the current and previous version due to the lack of knowledge.